RLBotPack/Diablo/diablo.cfg

```diff
@@ -6,18 +6,18 @@ looks_config = ./appearance.cfg
 python_file = ./diablo.py
 
 # Name of the bot in-game
-name = Diablo
+name = Diablo_dev
```

Comment:
Is this name change intentional?

Reply:
Nope, added the wrong config file apparently.